index.js

```diff
@@ -203,6 +203,21 @@ Metric.prototype._sendMetrics = function() {
   }, self.options.sendCallback);
 };
 
+/**
+ * Shuts down metric service by clearing any outstanding timer and sending any existing metrics
+ */
+Metric.prototype.shutdown = function() {
+  clearInterval(this._interval);
+  this._sendMetrics();
+};
+
+/**
+ * Gets whether outstanding metrics exist or not
```

Comment:
nit: let's add the jsdoc return type here.

Reply:
2237c49